### ISSUE TITLE
Kapton Tape Masked Pico W pin

### ISSUE THREAD

**Issue by baz4096** [V2RetroComputing/analog#18]
I've just noticed in one of the Tindie images, the Pico W RUN pin is masked with kapton tape, but the non-Wireless version isn't. 
I can't find any mention of masking any pins off, is this something I should be doing?

Masked pin:

https://cdn.tindiemedia.com/images/resize/zkVL8SAwADsTMNlKTqUq5UxlFvg=/p/full-fit-in/2400x1600/i/97799/products/2023-01-11T21%3A10%3A14.900Z-v2-analog-w.jpg?1673443949
https://cdn.tindiemedia.com/images/resize/zbUoMKf0rmjaOy2Pp3JLhiinn0Y=/p/full-fit-in/2400x1600/i/97799/products/2023-01-12T05%3A28%3A54.678Z-v2-analog-byop.jpg?1673472556

No masked pin:

https://cdn.tindiemedia.com/images/resize/YXppaIqB0wOAd4m_HjGg-x8-TWA=/p/full-fit-in/2400x1600/i/97799/products/2023-01-12T05%3A28%3A54.678Z-v2-analog-lc.jpg?1673472556

Thanks!

**Comment by Verault** [comment]
Great Question Baz4096. I too was going to ask about this same thing.

**Comment by dkgrizzly** [comment]
The original GERBERS for the board had the RESET line connected, which turned out to cause the RP2040 to boot too slow and reboot every time the Apple was reset. Isolating that pin allows the RP2040 to boot fast enough to catch the first writes to screen memory which is used to detect the machine type.

**Comment by Verault** [comment]
Just to clarify, the kapton tape is no longer needed on later revisions?

**Comment by baz4096** [comment]
As far as I can read from the commits, the Rev1 gerber was modified and committed Apr 17th as part of the repo reorganisation; the filename changed from "AppleII-Pico-F_Cu.gtl" to "V2-Analog-Rev1-F_Cu.gtl"

analog/GERBERS-Rev1/AppleII-Pico-F_Cu.gtl includes the reset track. (Available until 17th April)
analog-hardware/GERBERS-Rev1/V2-Analog-Rev1-F_Cu.gtl has the reset track removed. (From 17th April onwards)

The version of board I have includes the extra track. I've already soldered the Pico in place, so if I need to, I'll cut the trace where it emerges between Pico pins 12 & 13.

I'm still unsure if I need to isolate the reset pin, I guess I'll find out soon. My level shifters have arrived.

**Comment by Verault** [comment]
Thanks Baz4096. Please keep us informed on what you find.

**Comment by dkgrizzly** [comment]
Yes, if the trace is present, cut it. Otherwise you'll have issues with the autodetection and the RP2040 will reboot any time you hit ctrl-reset.
